RAGchain/benchmark/answer/metrics.py

```diff
@@ -106,3 +106,15 @@ def __init__(self):
     def retrieval_metric_function(self, knowledge: List[str], pred: str) -> float:
         score = self._token_f1_score(pred, "\n".join(knowledge))
         return score
+
+class METEOR(BasePassageAnswerMetric):
```

Comment:
meteor is answer metric. Inherit `BaseAnswerMetric` instead of `BasePassageAnswerMetric`